services/dashboard/routes/workers.go

```diff
@@ -0,0 +1,153 @@
+package routes
+
+import (
+	"encoding/base64"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+
+	"sensorbucket.nl/sensorbucket/internal/web"
+	"sensorbucket.nl/sensorbucket/pkg/api"
+	"sensorbucket.nl/sensorbucket/services/dashboard/views"
+)
+
+type WorkerPageHandler struct {
+	router chi.Router
+	client *api.APIClient
+}
+
+func CreateWorkerPageHandler(client *api.APIClient) *WorkerPageHandler {
+	handler := &WorkerPageHandler{
+		router: chi.NewRouter(),
+		client: client,
+	}
+	handler.SetupRoutes(handler.router)
+	return handler
+}
+
+func (h WorkerPageHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	h.router.ServeHTTP(w, r)
+}
+
+func (h *WorkerPageHandler) SetupRoutes(r chi.Router) {
+	r.Get("/", h.listWorkers())
+	r.Get("/create", h.createWorkerPage())
+	r.Post("/create", h.createWorker())
+	r.Get("/{id}", h.workerDetails())
+	r.Patch("/{id}", h.updateWorker())
+}
+
+func (h *WorkerPageHandler) listWorkers() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		page := &views.WorkerListPage{}
+		res, _, err := h.client.WorkersApi.ListWorkers(r.Context()).Execute()
```

Comment:
how does pagination work for the listWorkers? 

It's fixed in https://github.com/sensorbucket/SensorBucket/pull/52 so no need to edit here but is there something else going on because I'm not seeing the NextPage cursor being set